terraform/eks.tf

```diff
@@ -0,0 +1,97 @@
+// Production
+module "eks-production" {
+  source           = "terraform-aws-modules/eks/aws"
+  version          = "13.2.1"
+  cluster_name     = local.k8s_cluster_name
+  cluster_version  = "1.18"
+  subnets          = module.vpc.private_subnets
+  vpc_id           = module.vpc.vpc_id
+  write_kubeconfig = false
+  enable_irsa      = true
+  map_roles = [
+    {
+      rolearn  = aws_iam_role.kubectl.arn
+      username = aws_iam_role.kubectl.name
+      groups   = ["system:masters"]
```

Comment:
I likely just missed it but out of curiosity where is "system:masters" coming from?

Reply:
It's the default superuser group in k8s ([docs](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles)).

Reply:
oh that's cool